components/org.wso2.carbon.identity.application.authentication.handler.session/src/main/java/org/wso2/carbon/identity/application/authentication/handler/session/ActiveSessionsLimitHandler.java

```diff
@@ -147,7 +149,14 @@ protected void initiateAuthenticationRequest(HttpServletRequest request,
             Map<String, String> paramMap = new HashMap<>();
             paramMap.put(PROMPT_ID, context.getContextIdentifier());
             paramMap.put(SP_NAME, context.getServiceProviderName());
-            response.sendRedirect(FrameworkUtils.buildURLWithQueryParams(REDIRECT_URL, paramMap));
+            String redirectURL = FrameworkUtils.buildURLWithQueryParams(REDIRECT_URL, paramMap);
+            try {
+                redirectURL = ServiceURLBuilder.create().addPath(redirectURL).build().getAbsolutePublicURL();
+            } catch (URLBuilderException var4) {
+                throw new AuthenticationFailedException("Error while building tenant qualified url for context: "
+                        + REDIRECT_URL);
```

Comment:
exception object is not rethrown

Reply:
Fixed with 9b4e4a4cd3b6b1a3bae623fdd176ece989986c9b